### PR TITLE
tickets/DM-19782

### PR DESCRIPTION
**Pull request by fred3m**
Fix the two bugs identified by @yalsayyad and @laurenam, namely building a PSF image from different sized PSFs and skipping sources with no flux above the noise.

### CODE REVIEW

File: python/lsst/meas/extensions/scarlet/deblend.py [hunk 1]
```diff
@@ -50,6 +52,51 @@ def _getTargetPsf(shape, sigma=1/np.sqrt(2)):
...
+def _computePsfImage(self, position=None):
+    """Get a multiband PSF image
+    The PSF Kernel Image is computed for each band
+    and combined into a (filter, y, x) array and stored
+    as `self._psfImage`.
+    The result is not cached, so if the same PSF is expected
+    to be used multiple times it is a good idea to store the
+    result in another variable.
+
+    Note: this is a temporary fix during the deblender sprint.
```
Comment (yalsayyad) [inline python/lsst/meas/extensions/scarlet/deblend.py:63]: Need a ticket number for all TODOs

Comment (fred3m) [reply python/lsst/meas/extensions/scarlet/deblend.py:63]: Opened DM-19789

File: python/lsst/meas/extensions/scarlet/deblend.py [hunk 2]
```diff
@@ -431,7 +478,7 @@ def deblend(self, mExposure, sources):
             nchild = 0
             for k, source in enumerate(blend.sources):
                 py, px = source.pixel_center
-                if source.morph.sum() == 0 or source.sed.sum() == 0:
+                if source.morph.sum() == 0 or source.sed.sum() == 0 or source.skipped:
```
Comment (yalsayyad) [inline python/lsst/meas/extensions/scarlet/deblend.py:482]: Inline comment for what its checking for.

File: python/lsst/meas/extensions/scarlet/source.py [hunk 3]
```diff
@@ -21,14 +32,26 @@ def __init__(self, peak, scene, observations, bg_rms, bbox, obs_idx=0,
...
             # it as a PointSource
-            PointSource.__init__(self, sky_coord, scene, observations, symmetric, monotonic,
-                                 center_step, **component_kwargs)
+            try:
+                PointSource.__init__(self, sky_coord, scene, observations, symmetric, monotonic,
+                                     center_step, **component_kwargs)
+            except SourceInitError:
+                # There is really no flux to use for initializing this source,
+                # so just use an empty placeholder.
+                # TODO: In the future we should probably just strip blank
```
Comment (yalsayyad) [inline python/lsst/meas/extensions/scarlet/source.py:49]: Needs ticket number

Comment (fred3m) [reply python/lsst/meas/extensions/scarlet/source.py:49]: Opened DM-19790

**Pull request merged**